src/Concordium/Client/Commands.hs

```diff
@@ -2011,6 +2021,35 @@ identityShowARsCmd =
             (progDesc "Show anonymity revokers at a given block.")
         )
 
+poolCmds :: Mod CommandFields Cmd
+poolCmds =
+    command
+        "pool"
+        ( info
+            ( PoolCmd
+                <$> hsubparser
+                    poolStatusCmd
+            )
+            (progDesc "Commands for inspecting a validator pool or passive delegation.")
+        )
+
+poolStatusCmd :: Mod CommandFields PoolCmd
+poolStatusCmd =
+    command
+        "status"
+        ( info
+            ( PoolStatus
+                <$> optional (option auto (long "pool" <> metavar "POOL" <> help "Validator ID of pool. If not provided, status of passive delegation is queried."))
+                <*> optional
+                    ( strArgument
+                        ( metavar "BLOCK-HASH"
+                            <> help "Hash of the block to query (default: Query the best block)"
+                        )
+                    )
+            )
```

Comment:
I'd suggest making `POOL` an argument and `BLOCK-HASH` an optional parameter, so you would have `concordium-client pool status 10 --block <hash>` rather than `concordium-client pool status --pool 10 <hash>`.